DeAceroDLabs/marketplace-frontend
# Feature/dmp 435

## What is done

[[DMP-435] Empty cart](https://deacero.atlassian.net/browse/DMP-435)

## How is done

- Added to `Header` component tooltip for when cart is empty.

## How to test

1. Run the project via `npm start`
2. You should see the Home Screen. There you will see the cart icon on the screen.
3. Click on cart icon.
